portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```

Comment:
Agreed with what you mentioned, I would rename this to addRandomMovieQuote or something similar and replace instances of "greeting" with "quote" or "movieQuote" (see https://google.github.io/styleguide/jsguide.html#naming).